pom.xml

```diff
@@ -216,7 +216,7 @@
         <version.org.wildfly.openssl.wildfly-openssl-solaris-x86_64>${version.org.wildfly.openssl.natives}</version.org.wildfly.openssl.wildfly-openssl-solaris-x86_64>
         <version.org.wildfly.openssl.wildfly-openssl-windows-i386>${version.org.wildfly.openssl.natives}</version.org.wildfly.openssl.wildfly-openssl-windows-i386>
         <version.org.wildfly.openssl.wildfly-openssl-windows-x86_64>${version.org.wildfly.openssl.natives}</version.org.wildfly.openssl.wildfly-openssl-windows-x86_64>
-        <version.org.wildfly.security.elytron>1.9.0.CR4</version.org.wildfly.security.elytron>
+        <version.org.wildfly.security.elytron>1.9.0.CR5-SNAPSHOT</version.org.wildfly.security.elytron>
```

Comment:
Overall when submitting a PR please never include the dependency on the SNAPSHOT version of the component, it makes it impossible to coordinate the merging of this PR as first that needs to be undone.

Overall please just submit the changes relevant to your enhancement.